lib/index.ts

```diff
@@ -71,4 +71,18 @@ export const getJWTToken = (opts: JWTOptions): Promise<JWTResponse> => {
         });
     })
 
+}
+
+// support custom domains so that newly created scratch orgs don't have to wait for login servers at test.salesforce.com to propogate
+// jwt doesn't accept custom domains as audiences, but we can use them as the token endpoint
+
+const audienceTranslator = (audience: string) => {
+    if (audience === 'https://test.salesforce.com' || audience === 'https://login.salesforce.com') {
```

Comment:
should this check be case insensitive?   EG if I pass `https://test.Salesforce.com` I think this function will return `https://login.salesforce.com`, which could lead to some confusion

Reply:
yes, probably.